packages/serverless/src/domain/use-case/group.ts

```diff
@@ -1,254 +1,350 @@
-    const modelUser = new ModelUser();
+      const group = await modelGroup.findOne({ groupName }, { session });
 
-    const groupExist = (await modelGroup.findGroup(group, session)) !== null;
+      if (group) {
+        const modelUser = new ModelUser();
+        // Check list of user exist
+        if (await modelUser.areUsersExist(listUserName)) {
+          const modelUserGroup = new ModelUserGroup(databaseName);
+          const result = await modelUserGroup.addUserListToGroup(
+            listUserName,
+            groupName,
+            {
+              session,
+            }
+          );
 
-    if (groupExist) {
-      if (await modelUser.areUsersExist(users)) {
-        const modelUserGroup = new ModelUserGroup(databaseName);
-        const result = await modelUserGroup.removeUsersFromGroup(users, group, {
-          session,
-        });
+          return result.isOk();
+        }
 
-        return result.isOk();
+        throw Error('Some of user in list do not exist');
       }
 
-      throw Error('Some of users do not exist');
+      throw Error(`Group ${group} does not exist`);
     }
-
-    throw Error(`Group ${group} does not exist`);
+    throw Error('Only database owner allowed to add users to group');
   }
 
-  throw Error('Only database owner allowed to exclude users from group');
-}
+  /**
+   * Removes a list of users from a group based on the provided parameters.
+   *
+   * @param params - The parameters specifying the group and the users to remove, following the `TParamGroupAddListUser` type.
+   * @param session - (Optional) The MongoDB session for transactional queries.
+   * @returns A promise resolving to `true` if the users were removed successfully, otherwise `false`.
+   */
+  public static async removeListUserFromGroup(
+    params: TParamGroupAddListUser,
+    session?: ClientSession
+  ): Promise<boolean> {
+    const { databaseName, groupName, createdBy, listUserName } = params;
+    // Permission owner checking
+    if (await isDatabaseOwner(databaseName, createdBy, session)) {
+      // Initialize group model
+      const modelGroup = new ModelGroup(databaseName);
 
-async function renameGroup(
-  databaseName: string,
-  actor: string,
-  groupName: string,
-  newGroupName: string,
-  session?: ClientSession
-) {
-  if (await isDatabaseOwner(databaseName, actor, session)) {
-    const modelUserGroup = new ModelGroup(databaseName);
-    const group = await modelUserGroup.findGroup(groupName);
-    if (group) {
-      const result = await modelUserGroup.collection.updateOne(
-        {
-          groupName,
-        },
-        { $set: { groupName: newGroupName } }
-      );
-      return result && result.modifiedCount === 1;
+      const group = await modelGroup.findOne({ groupName }, { session });
+
+      if (group) {
+        const modelUser = new ModelUser();
+        // Check list of user exist
+        if (await modelUser.areUsersExist(listUserName)) {
```

Comment:
Must be `areListUserExist`, no plural please.

Reply:
Let keep it in this PR for only group. I do the refactor for User in next PR